README.md

```diff
@@ -1 +1,8 @@
-# gold-project
\ No newline at end of file
+# gold-project
+
+Main Functionality
```

Comment:
You can make "Main Functionality" a sub-header by prepending the text with "##".